### ISSUE TITLE
Request: open props tokens

### ISSUE THREAD

**Issue by Tylopilus** [tokencss/tokencss#10]
As Open props features a very good baseline and is actively maintained by @argyleink this would make a perfect fit.

**Comment by natemoo-re** [comment]
I totally agree! Not sure how easy it would be to keep in sync, but our preset is using _some_ tokens from Open Props. https://github.com/tokencss/tokencss/blob/main/packages/core/preset/token.config.json

**Comment by argyleink** [comment]
sync some props with https://unpkg.com/open-props/open-props.tokens.json maybe? latest json build 👍🏻 

i could also try and make an open-props preset for this? (like [figma tokens](https://github.com/argyleink/open-props/blob/main/build/to-figmatokens.js))

**Comment by alolalo** [comment]
@argyleink whatsup brother from another mother! 

Guess what I just realized? I think it would be awesome if I could be the "connector" of all the OpenProps tokens, by (un)democratic approval of all who think "naming things" is the hardest thing to do.

I would love to show some appreciation "under the hood" that TokenCSS solves; making CSS variables a pain in the ass to write out and construct like Sass, and making CSS more robust to read and write as a designer who likes to keep it simple.

Checking the links in adam's reply against the token.config.json files (plural), I can see two things; 

in TokenCSS you have two token.config.json files you can manage to "write" out your tokens. And second the "naming" of things not only needs to be linked, it needs to hold the same logic (like how TokensCSS uses 10 shades of color, where you might use 9 or less). This creates "underrepresented" values or "overrepresented values" unilaterally upon contact.

That overshoot/undershoot problem by system design can be solved with [B.O.U.M](https://www.followgu.us/boum). But that's the B-idea, I would rather just "hard-link" @argyleink OpenProps and understand his "way of thinking", so that "we" first get A-idea; linking and syncing tokens. 

Simply put; I would love to see how BOUM vs DTWG holds up in **keystrokes**, AND how it works with non-coders using design tooling. TokenCSS is a perfect ground imo, and OpenProps the perfect link for me. BOUM could connect;

1. color-scale steps
2. font-scale steps
3. font-size steps
4. font-weight steps
5. spacing-scale steps
6. **fluid-scale steps**
7. tshirt-sizing steps
8. ANY scale basically

So In short; by **first principle** I would love to create an API for design tokens, the "Dutch Way".
